bundles/org.openhab.binding.blink/src/main/java/org/openhab/binding/blink/internal/BlinkBindingConstants.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
```

Comment:
```suggestion
 * Copyright (c) 2010-2024 Contributors to the openHAB project
```

Please also update the others.

Reply:
If you pull main and rebase, this can help you with that:
```shell
mvn license:format
```
(from the repository root directory).